ui/apps/platform/src/Containers/Violations/ViolationsTablePage.tsx

```diff
@@ -66,7 +66,7 @@ function ViolationsTablePage(): ReactElement {
     const { isFeatureFlagEnabled } = useFeatureFlags();
     const isPlatformComponentsEnabled = isFeatureFlagEnabled('ROX_PLATFORM_COMPONENTS');
 
-    const [activeViolationStateTab, setActiveViolationStateTab] = useURLStringUnion(
+    const [selectedViolationStateTab, setSelectedViolationStateTab] = useURLStringUnion(
```

Comment:
Since `active` is a term we use for a Violation State, I thought we should rename this prefix.